app/uk/gov/hmrc/thirdpartydeveloperfrontend/controllers/support/HelpWithUsingAnApiController.scala

```diff
@@ -93,6 +93,20 @@ class HelpWithUsingAnApiController @Inject() (
     }
   }
 
+  override def otherValidation(flow: SupportFlow, extraData: List[ApiDefinition], form: Form[HelpWithUsingAnApiForm]): Form[HelpWithUsingAnApiForm] = {
+    def validate(fieldName: String) = {
+      val formFieldValue = form(fieldName).value.getOrElse("")
+      val matchesApiName = extraData.find(_.serviceName.value == formFieldValue).isDefined
+      if (matchesApiName) form else form.withGlobalError("Please choose a valid api name")
```

Comment:
Should `api` be `API` since it's an acronym and this is text on a web page, not a variable?